modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -258,6 +261,14 @@ export function useObjectFieldForm({
 			if (!settings.maximumFileSize && settings.maximumFileSize !== 0) {
 				errors.maximumFileSize = REQUIRED_MSG;
 			}
+			else if (settings.maximumFileSize > maxUploadRequestSize) {
+				errors.maximumFileSize = Liferay.Util.sub(
+					Liferay.Language.get(
+						'file-size-is-larger-than-maximum-upload-request-size-x-mb'
+					),
+					maxUploadRequestSize
+				);
+			}
```

Comment:
Hi @SelenaAungst, thansk for working on this.

I'm afraid this won't work properly because, from what I could see, `Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE` returns a value in `bytes` and `settings.maximumFileSize` returns a value in `megabytes`.

Also, I'm not sure if this is the appropriate error for this case. I think the error shown in the ticket is because the `maximumFileSize` exceeds the maximum value of a `long` in Java. Futhermore, I was not able to reproduce the issue. Could you please check again?

Thanks!

Reply:
Hi @SelenaAungst,

I talked to our QA and it seems that [LPS-150198](https://issues.liferay.com/browse/LPS-150198) is no longer reproducible. Another bug ticket has been created as you can see in [LPS-151863](https://issues.liferay.com/browse/LPS-151863).

Reply:
Hi @carolmariaabb,

I have started working on the new ticket for this bug. Underneath the input box for max file size on the edit attachment field page, it states "the maximum file size allowed is set by the 'Overall Maximum Upload Request Size' property in the 'Upload Servlet Request' configuration." I understand this as meaning the value that the user can input into the box must be smaller than the 'Overall Maximum Upload Request Size' which is stored as UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE. Therefore, the added validation provides the appropriate error message if the value entered by the user is larger than the upload max size (converted from bytes to megabytes this time). The value for 'Overall Maximum Upload Request Size' can be changed by the user in System Settings, but this value cannot be set larger than the maximum value of a long in Java, so the previous validation should account for that. Please let me know if my understanding is incorrect! 

Thank you!

Reply:
Hi @SelenaAungst, I will close this PR as you have already sent us another one. Thanks!